samples/hosting/generic-host/Core_7/GenericHost/GenericHost.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Worker">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
```

Comment:
why is this needed?

Reply:
It is not strictly needed but shows self-contained deployment according to https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/windows-service?view=aspnetcore-3.1&tabs=visual-studio#self-contained-deployment-scd

Reply:
does this sample need to be optimized for self-contained deployment?

Reply:
It was mentioned in the guidance so I figured I add it. I'm not opinionated enough to go one way or the other